Breadcrumbs.py

```diff
@@ -32,80 +38,169 @@ def get_row_indentation(points, view, tab_size, limit=1e20):
 def is_white_row(view, points):
   return all(view.substr(pt).isspace() for pt in reversed(points))
 
-def get_breadcrumb(view, points, limit):
+def get_breadcrumb(view, points, regex, separator):
   for pt in points:
     ch = view.substr(pt)
     if not ch.isspace():
-      return view.substr(sublime.Region(pt, min(view.line(pt).b, pt + limit))).strip()
+      linestring = view.substr(sublime.Region(pt, min(view.line(pt).b, pt + 500))).strip()
```

Comment:
It's not super easy to strip trailing whitespace in the regex and you don't really want to have to deal with this anyway. Which is why `trim_trailing_white_space_on_save` exists of course. So I opted to change it to `rstrip()` and let the regex deal with just the leading whitespace.